### PR TITLE
Fix bugs in features/aruba

### PR DESCRIPTION
**Pull request by maxmeyer**
## Summary

Fix CI errors.

## Details

This PR fixes errors on CI to make the build pass again.

## Motivation and Context

Make the build pass again.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (cleanup of codebase withouth changing any existing functionality)

## Checklist:

- [ ] I've added tests for my code
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by maxmeyer** [comment]
@mvz Feel free to push to this PR/branch.

File: features/04_aruba_api/core/expand_path.feature [hunk 1]
```diff
@@ -69,7 +69,7 @@ Feature: Expand paths with aruba
     RSpec.describe 'Expand path', :type => :aruba do
       let(:path) { '~/path/to/dir' }
 
-      it { expect(expand_path(path)).to match %r</home/[^/]+/path/to/dir> }
+      it { expect(expand_path(path)).to match %r</home/([^/]+/)+path/to/dir> }
```
Comment (mvz) [inline features/04_aruba_api/core/expand_path.feature]: I think the intent here was for `[^/]+` to match exacly a user name.

The correct fix is probably to remove the 'before 1.0.0' part of the feature and fix the 1.0.0 part to not set things up in the `before` section.

Comment (maxmeyer) [inline features/04_aruba_api/core/expand_path.feature]: No, I think, it was a "hack" I introduced. But you're right, the correct way was to remove that test. I missed the comment last night. Thanks.

**Comment by mvz** [comment]
Some of the remaining failures are due to Aruba not waiting for the running command to stop. @maxmeyer can you advise on what the desired behavior should be? I'm talking about cases where right now the cucumber scenarios have this:

```
  Scenario: Running aruba interactively

    This starts a pry console with "aruba"'s api loaded.

    When I run `bin/console` interactively
    And I type "exit"
    Then the exit status should be 0
```

The last step does not wait for any command to finish, unlike the old pre-1.0 behavior.

**Comment by maxmeyer** [comment]
@mvz 

* First of all we've got an issue for this behaviour: https://github.com/cucumber/aruba/issues/432
* I thought about making stopping a command more explicit in aruba 1.0.0 or at least in 2.0.0, with a cucumber step, but I think that need's to be discussed upfront
* Personally I would expect aruba to wait a couple of seconds before stopping the command and checking the status
* This needs to work both with interactive and non-interactive commands

> The last step does not wait for any command to finish, unlike the old pre-1.0 behavior.

Unfortunately I think I broke this step "a long time ago" without having issues on CI for a while. At least the code in several parts which are involved is the same for quite a long time.

What do you think about this?

**Comment by maxmeyer** [comment]
@mvz I forced pushed a rebased version of this branch. Just for you to know. I removed the `if`-statements along with the changed tests.

**Comment by maxmeyer** [comment]
@mvz I would like to merge this branch and then setup another one to capture the discussion around the problems with failing tests due to aruba not waiting for the command to finish.

**Comment by mvz** [comment]
@maxmeyer sounds like a good plan.

**Pull request merged**